jlm/hls/util/view.cpp

```diff
@@ -162,7 +174,10 @@ isForbiddenChar(char c)
 }
 
 std::string
-structural_node_to_dot(rvsdg::StructuralNode * structuralNode)
+structural_node_to_dot(
+    rvsdg::StructuralNode * structuralNode,
+    std::unordered_map<rvsdg::output *, std::string> & o_color,
+    std::unordered_map<rvsdg::input *, std::string> & i_color)
```

Comment:
Naming: `inputColors` and `outputColors`

Reply:
Changed